packages/cli/src/options/beaconNodeOptions/execution.ts

```diff
@@ -9,14 +9,14 @@ export type ExecutionEngineArgs = {
 };
 
 export function parseArgs(args: ExecutionEngineArgs): IBeaconNodeOptions["executionEngine"] {
-  let jwtSecretHex;
+  let jwtSecret;
```

Comment:
Totally subjective but this scoped let variables don't let me sleep at night hahaha

Same logic can be achieved with a ternary operator, and the whole thing looks cleaner type wise

Reply:
resolved in https://github.com/ChainSafe/lodestar/pull/3777/commits/46e723b0912b42392276d384059d5aadbcb0cc43 :+1: